Mlem/App/Views/Root/Tabs/Settings/AccessibilitySettingsView.swift

```diff
@@ -43,6 +44,20 @@ struct AccessibilitySettingsView: View {
             } header: {
                 Text("Non-Text Indicators")
             }
+                       
+            Section {
+                if #available(iOS 18, *) {
+                    NavigationLink(
+                        "Animated Avatars",
+                        value: .init(localized: animatedAvatars.label),
+                        fallbackValue: "",
+                        systemImage: Icons.playCircle,
+                        destination: .settings(.animatedAvatars)
+                    )
+                }
+            } header: {
+                Text("Reduce Motion")
+            }
```

Comment:
This hides the setting on iOS 17, but not the section header. Fix:

```suggestion
            if #available(iOS 18, *) {
                Section {
                    NavigationLink(
                        "Animated Avatars",
                        value: .init(localized: animatedAvatars.label),
                        fallbackValue: "",
                        systemImage: Icons.playCircle,
                        destination: .settings(.animatedAvatars)
                    )
                } header: {
                    Text("Reduce Motion")
                }
            }
```